### PR TITLE
Block info msg fix

### PR DESCRIPTION
**Pull request by gterzian**
Serialization part of https://github.com/massalabs/massa/issues/2884

### CODE REVIEW

**Comment by gterzian** [comment]
@damip Ready for review. This one basically just chunks outgoing "block info" messages to avoid sending to many at once. When receiving the messages we already check the length of it inside the deserializer...

**Comment by gterzian** [comment]
@AurelienFT Do you want to review?

**Review by AurelienFT — COMMENTED** [review]
I don't have the full scope of this change but it's complicated to follow here. I think we might consider splitting the code in another function.

File: massa-network-worker/src/node_worker.rs [hunk 1]
```diff
@@ -160,64 +193,80 @@ impl NodeWorker {
...
-                                            for operation_id in &operation_ids {
-                                                match op_store.get(operation_id) {
-                                                    Some(operation) => {
-                                                        wrapped_operation_serializer
-                                                            .serialize(operation, &mut res)?;
-                                                    }
-                                                    None => {
-                                                        return Err(NetworkError::MissingOperation)
+                                for replies in
+                                    reply_list.chunks(MAX_ASK_BLOCKS_PER_MESSAGE as usize)
```
[the remark this answers is not in the document]
Comment (damip) [reply massa-network-worker/src/node_worker.rs:197]: I think we should put all test-default and runtime constants in a separate crate and only include it in main and the tests that need them. That way we will solve the problem of people using them everywhere. I opened a followup: https://github.com/massalabs/massa/issues/2898

**Comment by gterzian** [comment]
Ok thanks for the review, I have replaced the constant with the use of the config... @AurelienFT

**Comment by AurelienFT** [comment]
> Ok thanks for the review, I have replaced the constant with the use of the config... Re-splitting it into another function, I don't really see how it would make it clearer, actually personally I think it might make it less so, and in any case it appears like a minor issue, so maybe lets just leave it like that for now? @AurelienFT

Ok ! no problem

**Review by AurelienFT — APPROVED** [review]

File: massa-network-worker/src/node_worker.rs [hunk 2]
```diff
@@ -7,7 +7,7 @@ use super::{
 use itertools::Itertools;
 use massa_logging::massa_trace;
 use massa_models::{
-    constants::{MAX_ASK_BLOCKS_PER_MESSAGE, MAX_ENDORSEMENTS_PER_MESSAGE, NODE_SEND_CHANNEL_SIZE},
+    constants::{MAX_ENDORSEMENTS_PER_MESSAGE, NODE_SEND_CHANNEL_SIZE},
```
Comment (damip) [inline massa-network-worker/src/node_worker.rs:10]: the code still uses constants, let's get rid of all of them

Comment (damip) [reply massa-network-worker/src/node_worker.rs:10]: opened a followup

**Pull request merged**